examples/abitest/tests/src/tests.rs

```diff
@@ -21,13 +21,16 @@ use oak::grpc;
 use oak_runtime::proto::manager::{
     ApplicationConfiguration, LogConfiguration, NodeConfiguration, WebAssemblyConfiguration,
 };
+use oak_tests::WasmEntrypointName;
 use serial_test_derive::serial;
 use std::collections::HashMap;
 
 // Constants for Node config names that should match those in the textproto
 // config held in ../../client/config.h.
 const FRONTEND_CONFIG_NAME: &str = "frontend-config";
+const FRONTEND_ENTRYPOINT_NAME: &str = "frontend_oak_main";
 const BACKEND_CONFIG_NAME: &str = "backend-config";
+const BACKEND_ENTRYPOINT_NAME: &str = "backend_oak_main";
 const LOG_CONFIG_NAME: &str = "logging-config";
```

Comment:
Could you confirm that this would allow in a future PR to unify `frontend-config` and `backend-config` into a single config with the two Rust crates merged into one, and exporting two different entry points?